recipes/libucl/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import CMake, ConanFile, tools
+import functools
+
+required_conan_version = ">=1.33.0"
+
+
+class LibuclConan(ConanFile):
+    name = "libucl"
+    description = "Universal configuration library parser "
```

Comment:
trailing space

Reply:
Thanks for noticing.
```suggestion
    description = "Universal configuration library parser"
```